pkg/server/gateway.go

```diff
@@ -45,7 +45,7 @@ type GatewayConfig struct {
 	TLS     *TLSConfig
 }
 
-func NewGateway(address string, refresh time.Duration, caFile, certificateFile, keyFile string) (*KiamGateway, error) {
+func NewGateway(address string, refresh time.Duration, waitForBalancer time.Duration, caFile, certificateFile, keyFile string) (*KiamGateway, error) {
```

Comment:
Could we pass the `context.Context` through as the first arg to `NewGateway`, rather than the wait duration.